### PR TITLE
Ensures presence of answers

### PR DESCRIPTION
**Pull request by benlovell**

### CODE REVIEW

**Comment by alexwllms** [comment]
:+1:

File: spec/forms/pre_qualification_form_spec.rb [hunk 1]
```diff
@@ -3,32 +3,33 @@
...
-      question_4: '1'
-    } end
+    let(:params) do
+      {
+        question_1: '1',
+        question_2: '1',
+        question_3: '1',
+        question_4: '1'
+      }
+    end
```
Comment (rnalexander) [inline spec/forms/pre_qualification_form_spec.rb]: Why would you collapse the other ones but leave this one expanded?

**Comment by jongilbraith** [comment]
:+1:

File: spec/forms/pre_qualification_form_spec.rb (hunk 1 above)
Comment (benlovell) [inline spec/forms/pre_qualification_form_spec.rb]: Because it would have made an unsightly long line.

**Pull request merged**